provisioning/roles/sp_mini_7_setup_init/tasks/main.yml

```diff
@@ -9,29 +9,35 @@
     dest: "/etc/init.d"
     mode: 0755
   with_items:
-    - kibana4_init
     - snowplow_mini_control_plane_api
-    - snowplow_stream_collector
-    - snowplow_stream_enrich
-    - snowplow_elasticsearch_loader_good
-    - snowplow_elasticsearch_loader_bad
-    - iglu_server_0.2.0
     - caddy_init
-    - nsqd_init
-    - nsqlookupd_init
 
 - name: Configure for inits for calling at boot time
   become: yes
   shell: "update-rc.d {{item}} defaults"
   with_items:
-    - kibana4_init
     - snowplow_mini_control_plane_api
-    - snowplow_stream_collector
-    - snowplow_stream_enrich
-    - snowplow_elasticsearch_loader_good
-    - snowplow_elasticsearch_loader_bad
-    - iglu_server_0.2.0
-    - elasticsearch
     - caddy_init
-    - nsqd_init
-    - nsqlookupd_init
+
+- name: Start UI and Control Plane
+  become: yes
+  shell: service snowplow_mini_control_plane_api start && service caddy_init start
+
+- name: Increase mmap count to recommended 262144 for Elasticsearch
+  become: yes
+  shell: echo "vm.max_map_count=262144" >> /etc/sysctl.conf && service procps start
```

Comment:
this isn't in the container why would it affect es?

Reply:
I think it is because containers share the same kernel as host OS.